kubeluigi/k8s.py

```diff
@@ -170,11 +171,14 @@ def print_pod_logs(job: V1Job, pod: V1PodSpec):
             logger.warning("error while fetching pod logs :" + logs_prefix)
             logger.exception(e)
             raise e
-
-    stream = get_pod_log_stream(pod)
-    for i in stream:
-        l = logs_prefix + ": " + i
-        logger.info(l)
+    
+    try:
+        stream = get_pod_log_stream(pod)
+        for i in stream:
+            l = logs_prefix + ": " + i
+            logger.info(l)
+    except urllib3.exceptions.ProtocolError:
```

Comment:
can we retry?
Should we swallow this error?

Reply:
It is not very important since we have another way to view the logs
I also changed its level to "debug" so that it does not show up in normal runs at all